packages/wallet/ui/src/components/ProvisionDialog.jsx

```diff
@@ -105,8 +161,14 @@ const ProvisionDialog = ({ onClose, open, address, href, keplrConnection }) => {
       </DialogTitle>
       <DialogContent>
         {content}
+        {provisionPoolLow && (
+          <DialogContentText sx={{ pt: 2 }} color="error">
+            Unable to create a Smart Wallet because the provision pool balance
+            is too low.
```

Comment:
I bet users are going to have a hard time making sense of this.
```suggestion
            The pool of funds to provision smart wallets is too small at this time.
```

Reply:
Done. This is still probably something they'll have to ask about in our social channels regardless, but hopefully no one runs into this at all in any regular circumstance.